internal/web_package/injector.js

```diff
@@ -51,18 +63,33 @@ function main(params, read = fs.readFileSync, write = fs.writeFileSync, timestam
     throw ('No <head> tag found in HTML document');
   }
 
+  /**
+   * Trims the longest prefix from the path
+   */
+  function relative(execPath) {
```

Comment:
Node also exposes `relative` in the `path` module.

Reply:
yeah, and it has weird cross-platform behavior too